drivers/platform/aducm3029/irq_extra.h

```diff
@@ -93,6 +95,17 @@ enum irq_mode {
 	IRQ_LOW_LEVEL		= ADI_XINT_IRQ_LOW_LEVEL
 };
 
+/**
+ * @struct rtc_irq_config
+ * @brief RTC interrupt configuration routine.
+ */
+struct rtc_irq_config {
+	/** RTC driver handler */
+	struct rtc_desc *rtc_handler;
+	/** Active interrupts OR'ed together */
+	uint32_t active_interrupts;
```

Comment:
I think you should define what values can be used for this parameter: 
Like 
```
#define RTC_ALARM                ADI_RTC_ALARM_INT
#define RTC_MOD60ALM        ADI_RTC_MOD60ALM_INT
#define RTC_ISO_DONE_INT    ADI_RTC_ISO_DONE_INT
```
Defining just what can be used with this driver

Reply:
Would this add anything or just be a translation from the DFP driver to this one?

Reply:
In order to not include the DFP from the appilcation and include only irq_extra.h

Reply:
In order to use no-os or dfp driver, not both